(it was a http.get which i forgot to handle "error" events for, in case anyone else were curious)